packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_maps_flutter_ios.dart

```diff
@@ -580,6 +614,37 @@ class GoogleMapsFlutterIOS extends GoogleMapsFlutterPlatform {
     GoogleMapsInspectorPlatform.instance =
         GoogleMapsInspectorIOS((int mapId) => _channel(mapId));
   }
+
+  /// Parses cluster data from dynamic json objects and returns [Cluster] object.
+  /// Used by `cluster#onTap` method call handler and inspectors [getClusters] response parser.
+  static Cluster parseCluster(
+      String clusterManagerIdString,
+      Object positionObject,
+      Map<dynamic, dynamic> boundsMap,
```

Comment:
`dynamic` is very strongly discouraged in all new code; please use `Object` or `Object?` as appropriate.